landing_page_backend/contacts/management/commands/weekly_reports.py

```diff
@@ -0,0 +1,22 @@
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+
+from contacts.models import Contact
+from contacts.utils import write_data_to_sfv_file
+from landing_page_backend.settings import EMAIL_HOST_USER
+
+
+class Command(BaseCommand):
+    help = 'Weekly reports'
+
+    def handle(self, *args, **options):
+        contacts = Contact.objects.all().values_list('name', 'email', 'number', 'message')
+        for contact in contacts:
+            write_data_to_sfv_file(contact)
+
+        message = 'Weekly reports'
+        subject = 'This is weekly reports'
+        email = EmailMessage(subject, message, EMAIL_HOST_USER, ['gorbunovmy.a@gmail.com'])
+        file = open('contacts-2022-09-01.csv', 'r')
```

Comment:
File name should be generated automatically based on the current date. If I will run this script in 2 weeks, it will cause issue.